src/app/components/bitcoin-contract-entry-point/bitcoin-contract-entry-point-layout.tsx

```diff
@@ -0,0 +1,63 @@
+import { Flex, StackProps } from '@stacks/ui';
+import { forwardRefWithAs } from '@stacks/ui-core';
+import { CryptoAssetSelectors } from '@tests/selectors/crypto-asset.selectors';
+import { HStack } from 'leather-styles/jsx';
+
+import { Money } from '@shared/models/money.model';
+
+import { formatBalance } from '@app/common/format-balance';
+import { ftDecimals } from '@app/common/stacks-utils';
+import { Flag } from '@app/components/layout/flag';
+import { Tooltip } from '@app/components/tooltip';
+import { Caption, Text } from '@app/components/typography';
+
+import { SmallLoadingSpinner } from '../loading-spinner';
+
+interface BitcoinContractEntryPointLayoutProps extends StackProps {
+  balance: Money;
+  caption: string;
+  icon: React.JSX.Element;
+  usdBalance?: string;
+  isLoading?: boolean;
+  onClick: () => void;
+}
+export const BitcoinContractEntryPointLayout = forwardRefWithAs(
```

Comment:
Feel free to leave this if you don't feel comfortable taking on yourself, but we're moving away from StacksUI in favour of Panda via `leather-styles/jsx`, which has a suite of similar components, `Flex`, `Box`, etc. 

Would be great to replace StacksUI with those

Reply:
Removed it where it had the same attributes as in stacks/ui